app/src/main/res/values/themes.xml

```diff
@@ -16,6 +16,7 @@
         <item name="dragHandleColorHamdan">#aaaaaa</item>
         <item name="tabBackgroundColorHamdan">#ffffff</item>
         <item name="selectedTabColorHamdan">#aaaaaa</item>
+        <item name="foregroundColorHamdan">#aaaaaa</item>
```

Comment:
Same as color above?

Reply:
it will be different with proper themeing, for now its a copy essentially